docs/src/configuration/services/rabbitmq.md

```diff
@@ -66,36 +66,51 @@ highlight=python
+(The specific way to inject configuration into your application varies.
+Consult your application or framework's documentation.)
 
 ## Connecting to RabbitMQ
 
 ### From your local development environment
 
-For debugging purposes, it's sometimes useful to be able to directly connect to a service instance. You can do this using SSH tunneling. To open a tunnel, log into your application container like usual, but with an extra flag to enable local port forwarding:
+For debugging purposes, it's sometimes useful to connect directly to a service with an SSH tunnel.
+To open a tunnel, follow these steps.
 
-```bash
-ssh -L 5672:rabbitmqqueue.internal:5672 <projectid>-<branch_ID>@ssh.eu.platform.sh
-```
+1. Get the username and password to the service by checking your app's relationships:
 
-Within that SSH session, use the following command to pretty-print your relationships. This lets you see which username and password to use, and you can double check that the remote service's port is 5672.
+   ```bash
+   platform ssh 'echo $PLATFORM_RELATIONSHIPS | base64 --decode | jq .'
+   ```
 
-```bash
-php -r 'print_r(json_decode(base64_decode($_ENV["PLATFORM_RELATIONSHIPS"])));'
-```
+2. Open a tunnel with port forwarding:
 
-If your service is running on a different port, you can re-open your SSH session with the correct port by modifying your `-L` flag: `-L 5672:rabbitmqqueue.internal:<remote port>`.
+   ```bash
+   platform tunnel:single -g
+   ```
 
-Finally, while the session is open, you can launch a RabbitMQ client of your choice from your local workstation, configured to connect to `localhost:5672` using the username and password you found in the relationship variable.
+3. Launch a RabbitMQ client of your choice configured to connect to the location from step 2
+   with the username and password from step 1.
 
 ### Access the management plugin  (Web UI)
-In case you want to access the browser-based UI, you have to use an SSH tunnel. To open a tunnel, log into your application container like usual, but with an extra flag to enable local port forwarding:
 
-```bash
-ssh -L 15672:rabbitmqqueue.internal:15672 <projectid>-<branch_ID>@ssh.eu.platform.sh
-```
+To access the browser-based UI, use an SSH tunnel.
+To open a tunnel, follow these steps.
+
+1. SSH into your app container with a flag for local port forwarding:
+
+   ```bash
+   ssh $(platform ssh --pipe) -L 15672:rabbitmqqueue.internal:15672
+   ```
+
+2. Within that SSH session, get the username and password.
+   Use the following command to see your app's relationships:
+
+   ```bash
+   echo "$PLATFORM_RELATIONSHIPS" | base64 --decode | jq .
+   ```
 
-After you successfully established a connection, you should be able to open `http://localhost:15672` in your browser. You'll find the credentials like mentioned above.
+3. Open `http://localhost:15672` in your browser.
```

Comment:
We should test for rabbitmq specifically, but this again is the same as using the tunnel:single command

Reply:
The tunnel:single command seems to create a amqp connection string, but it didn't seem to create a link to the web UI. I didn't see a flag that would work for that.